webapp/plugins/i18n.js

```diff
@@ -22,14 +22,26 @@ export default ({ app, req, cookie, store }) => {
     if (process.server) return
 
     const newLocale = mutation.payload.locale
-    const currentLocale = await app.$cookies.get(key)
+    let isCookie = true
+    let currentLocale = await app.$cookies.get(key)
+    if (!currentLocale) {
+      isCookie = false
+      currentLocale = navigator.language.split('-')[0] // get browser language
+    }
     const isDifferent = newLocale !== currentLocale
 
-    if (!isDifferent) {
+    // cookie has to be set, otherwise Cypress test does not work
+    if (isCookie && !isDifferent) {
```

Comment:
```suggestion
    if (isCookie && !isDifferent) {
```
if (isCookie && isDifferent) {

Reply:
Done

Thanks for the idea @ogerly 🙏🏼